shell/platform/windows/platform_handler_unittests.cc

```diff
@@ -22,9 +21,11 @@ using ::testing::_;
 static constexpr char kChannelName[] = "flutter/platform";
 
 static constexpr char kGetClipboardDataMethod[] = "Clipboard.getData";
+static constexpr char kHasStringsClipboardMethod[] = "Clipboard.hasStrings";
 static constexpr char kSetClipboardDataMethod[] = "Clipboard.setData";
 
 static constexpr char kTextPlainFormat[] = "text/plain";
+static constexpr char kFakeContentType[] = "text/madeupcontenttype";
```

Comment:
This fake content type must begin with "text" even though it's supposed to fail.  If it doesn't, then rapidjson can't parse the json at all.